src/compat/db_vdb.c

```diff
@@ -581,6 +581,14 @@ db_compile_statement_local (DB_SESSION * session)
   /* forget about any previous parsing errors, if any */
   pt_reset_error (parser);
 
+  // @server
+  pt_check_server_extension (parser, statement);
```

Comment:
I think "pt_check_" only checks if it is a server_extension.
However, pt_check_server_extension() calls pt_convert_server() and changes the statement to server_extension.

So I want it to be a clear name like "pt_convert_" rather than "pt_check_".

Reply:
The function logic is convert, so I keep this function name.

Reply:
It would be better that the routine where the query is rewritten is placed in mq_translate() for consistency. And better to make the name clear like rewrite_for_dblink.

Reply:
Ok, I will fixed as your comments.